packages/salesforcedx-vscode-apex/package.json

```diff
@@ -174,6 +174,7 @@
     "expand-home-dir": "0.0.3",
     "find-java-home": "0.2.0",
     "path-exists": "3.0.0",
+    "shelljs": "^0.8.1",
```

Comment:
This is making the circle ci build fail. Take a look at the stack trace to see what needs to be changed.

Reply:
Thanks, looking into it now